local-development/docker-compose.yml

```diff
@@ -20,22 +20,19 @@ services:
         tag: fairspace-keycloak-postgres
 
   fairspace-keycloak:
-    image: jboss/keycloak:12.0.4
+    image: keycloak/keycloak:latest
```

Comment:
better to specify the version, otherwise if the next one can have breaking changes, the default setup will stop working
you were testing with 20.0.3, right?

Reply:
Well let's discuss during standup, I actually like it when it breaks so we can fix it. We should try to keep up to date with tooling, especially with the frontend security barrier. It will break locally first, not at a customer. If somehow we can't fix a breaking change we can always make it fixed.